src/options.ts

```diff
@@ -321,6 +321,13 @@ export async function resolveOptions(options: Options): Promise<{
       } = subOptions
 
       outDir = path.resolve(outDir)
+      if (outDir === process.cwd()) {
+        throw new Error(
+          'Output directory cannot be the same as the current working directory. ' +
+            `Please specify a different watch directory using ${blue`watch`} option,` +
```

Comment:
Only throw when `clean` or `watch` is enabled, also error message should be updated.

Reply:
The error message has been adjusted according to the previous one, but I’m unsure if it’s appropriate.